### PR TITLE
Dutch giftcards

### PR DESCRIPTION
**Pull request by JoostMollie**
** Checklist **
- [x] All icons have a corresponding entry in `db/payment_icons.yml`
- [x] I have followed the icon guidelines detailed in the [CONTRIBUTING.md](https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md) file
- [x] I have optimized the icon with [SVGO](https://jakearchibald.github.io/svgomg/)
- [x] I am confident that all icons are clear and easy to read/understand
- [x] I have provided a link to the brand icon’s brand guidelines whenever possible.

If this pull request is not adding new icons, you can remove this checklist.

### CODE REVIEW

File: db/payment_icons.yml [hunk 1]
```diff
@@ -994,3 +994,27 @@
   name: nelo
   label: Nelo
   group: other
+-
+  name: vvv_giftcard
+  label: VVV Cadeaukaart
+  group: other
+  -
```
Comment (larouxn) [inline db/payment_icons.yml:1001]: We need to move these `-` back to align with all other previous `-`.

**Review by larouxn — COMMENTED** [review]
👋 Hello @JoostMollie, can we update these icons to better align with all other icons' shape and such? There's a template over in the [contribution guide](https://github.com/activemerchant/payment_icons/blob/dc6de5046bf9afac185e5b54983eedd76b0a08a6/CONTRIBUTING.md#template).
Screen Shot 2021-09-16 at 10 14 15

File: db/payment_icons.yml (hunk 1 above)
Comment (JoostMollie) [reply db/payment_icons.yml]: Should be fixed now.. :)

**Comment by JoostMollie** [comment]
I have made changes to the icons.. Should now match the dimensions of the template.

File: app/assets/images/payment_icons/nationalebioscoopbon.svg [hunk 2]
```diff
@@ -0,0 +1,31 @@
+<svg xmlns="http://www.w3.org/2000/svg" role="img" viewBox="0 0 38 24" width="38" height="24" aria-labelledby="pi-nationalebioscoopbon">
```
Comment (tauthomas01) [inline app/assets/images/payment_icons/nationalebioscoopbon.svg:1]: Can we remove all instances of `id`? For example, these don't seem to be needed:

`id="Logo"`
`id="Shape"`

File: app/assets/images/payment_icons/nationaleentertainmentcard.svg [hunk 3]
```diff
@@ -0,0 +1,22 @@
+<svg fill="none" viewBox="0 0 38 24" width="38" height="24" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```
Comment (tauthomas01) [inline app/assets/images/payment_icons/nationaleentertainmentcard.svg:1]: Same comments apply about the `id` attributes.
Also the borders have some pixels issues as well as the radius
[alt](https://screenshot.click/16-05-nug4x-l7df1.png)

You can follow this [example icon](https://github.com/activemerchant/payment_icons/blob/master/app/assets/images/payment_icons/visa.svg) as a reference.

**Review by tauthomas01 — CHANGES_REQUESTED** [review]
Hi,

I left some comments.

Also one comment that applies to every icons: can we please put all code in 1 single line?

Please let me know if there are any questions.

File: app/assets/images/payment_icons/nationaleentertainmentcard.svg (hunk 3 above)
Comment (JoostMollie) [reply app/assets/images/payment_icons/nationaleentertainmentcard.svg]: Fixed pixel issues. I left the ID's that are used as a reference for the mask/gradients.

File: app/assets/images/payment_icons/nationalebioscoopbon.svg (hunk 2 above)
Comment (JoostMollie) [reply app/assets/images/payment_icons/nationalebioscoopbon.svg]: Done.

**Comment by JoostMollie** [comment]
@tauthomas01 I fixed the issues.. let me know if there is anything else!

**Review by tauthomas01 — APPROVED** [review]
This looks good

**Comment by tauthomas01** [comment]
The CI outputs the following error:

`The 'kunstencultuurcadeaukaart' SVG file does not have the appropriate <title> value`

**Comment by JoostMollie** [comment]
> The CI outputs the following error:
> 
> `The 'kunstencultuurcadeaukaart' SVG file does not have the appropriate <title> value`

Not sure what is going on here.. the title looked good to me. I did update the yml file for this entry to align the label with what is between the <title> brackets.. otherwise I would not know what would cause this error.

**Comment by larouxn** [comment]
:wave: @JoostMollie, looks like we need to syncronize the `db/payment_icons.yml` labels with the SVG titles or the other way around. For example I'm seeing the following error. ([logs](https://github.com/activemerchant/payment_icons/pull/485/checks?check_run_id=3657729468))
```
# Running:

........F

Failure:
PaymentIconTest#test_Every_payment_SVG_meets_accessibility_requirements [/home/larouxn/src/github.com/activemerchant/payment_icons/test/unit/payment_icon_test.rb:78]:
{:message=>"The 'nationalebioscoopbon' SVG file does not have the appropriate <title> value"}.
Expected: "Nationale Bioscoopbon"
  Actual: "Nationale Bioscoop Bon"
```

We can fix this by setting both the yml label and SVG title to the same value, either `Nationale Bioscoop Bon` or `Nationale Bioscoopbon`, as currently they are different.

```
# Running:

.........

Finished in 0.133399s, 67.4667 runs/s, 40217.6458 assertions/s.
9 runs, 5365 assertions, 0 failures, 0 errors, 0 skips
```

**Comment by JoostMollie** [comment]
> 👋 @JoostMollie, looks like we need to syncronize the `db/payment_icons.yml` labels with the SVG titles or the other way around. For example I'm seeing the following error. ([logs](https://github.com/activemerchant/payment_icons/pull/485/checks?check_run_id=3657729468))
> 
> ```
> # Running:
> 
> ........F
> 
> Failure:
> PaymentIconTest#test_Every_payment_SVG_meets_accessibility_requirements [/home/larouxn/src/github.com/activemerchant/payment_icons/test/unit/payment_icon_test.rb:78]:
> {:message=>"The 'nationalebioscoopbon' SVG file does not have the appropriate <title> value"}.
> Expected: "Nationale Bioscoopbon"
>   Actual: "Nationale Bioscoop Bon"
> ```
> 
> We can fix this by setting both the yml label and SVG title to the same value, either `Nationale Bioscoop Bon` or `Nationale Bioscoopbon`, as currently they are different.
> 
> ```
> # Running:
> 
> .........
> 
> Finished in 0.133399s, 67.4667 runs/s, 40217.6458 assertions/s.
> 9 runs, 5365 assertions, 0 failures, 0 errors, 0 skips
> ```

Everything should be aligned now!

**Comment by larouxn** [comment]
:wave: @JoostMollie, the changes make sense to me. Can we rebase this PR on latest master to resolve the merge conflicts? Once we've done that we can run CI, confirm everything's good, and merge.

**Comment by JoostMollie** [comment]
Should be ok now I think :)

**Comment by larouxn** [comment]
👋 Hey @JoostMollie, thanks for the rebase. That part looks good. Looks like we just need to fix an ID in the `vvv` icon.
https://github.com/activemerchant/payment_icons/pull/485/checks?check_run_id=3715292440

```
Failure:
PaymentIconTest#test_Every_payment_SVG_meets_accessibility_requirements [/home/runner/work/payment_icons/payment_icons/test/unit/payment_icon_test.rb:89]:
{:message=>"The 'a' ID should be pi-vvv_giftcard-a (missing 'pi-' prefix)"}.
Expected /pi-(.*)/ to match "a".

rails test /home/runner/work/payment_icons/payment_icons/test/unit/payment_icon_test.rb:61

...........

Finished in 0.858207s, 13.9826 runs/s, 6078.9523 assertions/s.
12 runs, 5217 assertions, 1 failures, 0 errors, 0 skips
```

**Comment by JoostMollie** [comment]
@larouxn I've taken a look, but the check is mentioning a `mask` that has an ID needed for reference. So this ID is not for accessibility purposes. The ID's are correct in the `svg` element as well as the `title` element.

Let me know what the best way to move forward will be.

**Comment by tauthomas01** [comment]
> @larouxn I've taken a look, but the check is mentioning a `mask` that has an ID needed for reference. So this ID is not for accessibility purposes. The ID's are correct in the `svg` element as well as the `title` element.
> 
> Let me know what the best way to move forward will be.

I assume the tests are checking every `id` attributes and make sure they have  a `pi-` prefix, regardless if it's used for accessibility or not. Maybe could you give it a try?

**Comment by JoostMollie** [comment]
Ok I have changed it.. :)

**Comment by tauthomas01** [comment]
It seems to be working. We'll need to apply for the filter (same icon) according to the error message `#filter0_d`

**Comment by JoostMollie** [comment]
Ok I have updated them all now..

**Comment by larouxn** [comment]
Looks like `app/assets/images/payment_icons/nationalebioscoopbon.svg` still has some `id=""` without `pi-`.

```
Failure:
PaymentIconTest#test_Every_payment_SVG_meets_accessibility_requirements [/home/runner/work/payment_icons/payment_icons/test/unit/payment_icon_test.rb:89]:
{:message=>"The 'a' ID should be pi-nationaleentertainmentcard-a (missing 'pi-' prefix)"}.
Expected /pi-(.*)/ to match "a".
```
https://github.com/activemerchant/payment_icons/pull/485/checks?check_run_id=3728457212

**Comment by JoostMollie** [comment]
Ok I double checked all of them and updated the ID's for nationale entertainment card. The rest looks all ok. Should be good now (at least for the ID's :P )

**Comment by larouxn** [comment]
:wave: @JoostMollie, I think we just need to update `nationaleentertainmentcard` so it contains a `role` attribute and we should be good to go.

```svg
<svg fill="none" viewBox="0 0 38 24" width="38" height="24" role="img" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" aria-labelledby="pi-nationaleentertainmentcard"><title id="pi-nationaleentertainmentcard">Nationale Entertainment Card</title><radialGradient id="pi-nationaleentertainmentcard-a" cx=".5" cy=".5" gradientTransform="matrix(62.5 0 0 62 -31.344 -19.375)" gradientUnits="userSpaceOnUse" r=".5"><stop offset="0" stop-color="#12c6ff" stop-opacity=".5"/><stop offset="1" stop-opacity=".5"/></radialGradient><linearGradient id="pi-nationaleentertainmentcard-b"><stop offset="0" stop-color="#ff0202"/><stop offset=".15" stop-color="#ff9c01"/><stop offset=".32" stop-color="#fff400"/><stop offset=".475" stop-color="#7ed21c"/><stop offset=".649" stop-color="#b8dbf1"/><stop offset=".817" stop-color="#7f2b89"/><stop offset="1" stop-color="#dd0085"/></linearGradient><linearGradient id="pi-nationaleentertainmentcard-c" gradientTransform="matrix(22.4353 0 0 2.09297 .057 -.016)" gradientUnits="userSpaceOnUse" x2="1" xlink:href="#pi-nationaleentertainmentcard-b"/><linearGradient id="pi-nationaleentertainmentcard-d" gradientTransform="matrix(24.6209 0 0 2.7252 -5.046 -.298)" gradientUnits="userSpaceOnUse" x2="1" xlink:href="#pi-nationaleentertainmentcard-b"/><path fill="#000" d="M35 0H3C1.3 0 0 1.3 0 3v18c0 1.7 1.4 3 3 3h32c1.7 0 3-1.3 3-3V3c0-1.7-1.4-3-3-3z"/><path fill="url(#pi-nationaleentertainmentcard-a)" d="M35 0H3C1.3 0 0 1.3 0 3v18c0 1.7 1.4 3 3 3h32c1.7 0 3-1.3 3-3V3c0-1.7-1.4-3-3-3z"/><g clip-rule="evenodd" fill-rule="evenodd"><path d="M0 0v2.072h1.407v-.381H.425v-.493h.891V.83h-.89V.378h.981V0zm3.304 0h-.415v1.31L1.974 0h-.405v2.072h.415V.692l.979 1.38h.34zm.114.378h.665v1.694h.435V.378h.669V0H3.418zM5.298 0v2.072h1.407v-.381h-.982v-.493h.89V.83h-.89V.378h.982V0zM7.52 1.3h-.226v.772h-.428V0h.854c.435 0 .725.233.725.658 0 .33-.182.54-.472.611l.546.803h-.486zM7.295.367v.577h.399c.216 0 .327-.115.327-.287 0-.175-.112-.29-.328-.29zm1.158.01V0h1.768v.378h-.668v1.694h-.436V.378zM10.764 0h.429l.793 2.072h-.439l-.186-.5h-.776l-.182.5H9.97zm.21.52l-.254.688h.506zm1.52-.52h-.432v2.072h.432zm1.957 0h-.415v1.31L13.122 0h-.405v2.072h.416V.692l.978 1.38h.341zm1.438 2.072L16.392.72v1.353h.409V0h-.506l-.55 1.461L15.19 0h-.513v2.072h.405V.72l.513 1.353zM17.027 0v2.072h1.407v-.381h-.982v-.493h.89V.83h-.89V.378h.982V0zm3.303 0h-.415v1.31L19.001 0h-.405v2.072h.415V.692l.978 1.38h.341zm.115.378h.665v1.694h.435V.378h.668V0h-1.768z" fill="url(#pi-nationaleentertainmentcard-c)" transform="matrix(1.15 0 0 1.15 5.462 11.787)"/><path d="M1.735.034H1.32v1.31L.405.033H0v2.072h.415V.726l.979 1.38h.34zm1.418 0h.428l.793 2.072h-.438l-.186-.5h-.776l-.182.5H2.36zm.209.52l-.253.688h.506zM4.675.411h.665v1.694h.435V.412h.669V.034H4.675zm2.86-.378h-.433v2.072h.432zm2.629 1.036c0 .617-.405 1.07-.982 1.07-.578 0-.98-.453-.98-1.07 0-.621.402-1.07.98-1.07.577 0 .982.449.982 1.07zm-.436 0c0-.405-.216-.685-.546-.685-.328 0-.54.28-.54.685s.212.685.54.685c.33 0 .546-.28.546-.685zM12.574.034h-.415v1.31l-.915-1.31h-.405v2.072h.415V.726l.979 1.38h.34zm1.418 0h.428l.793 2.072h-.438l-.186-.5h-.776l-.183.5h-.432zm.209.52l-.253.688h.506zm1.637 1.552h1.397v-.381h-.965V.034h-.432zM17.924.034v2.072h1.408v-.381h-.983v-.493h.891V.864h-.89V.412h.982V.034z" fill="url(#pi-nationaleentertainmentcard-d)" transform="matrix(1.15 0 0 1.15 8.912 8.05)"/><path d="M20.7 16.468c0 .73.474 1.23 1.11 1.23.563 0 .908-.33.998-.811h-.493c-.082.221-.23.369-.505.369-.38 0-.605-.338-.605-.788 0-.454.225-.788.605-.788.276 0 .423.147.505.365h.493c-.09-.478-.435-.808-.998-.808-.636 0-1.11.501-1.11 1.23zm3.54-1.192h.493l.912 2.383h-.504l-.214-.574h-.893l-.21.574h-.496zm.24.598l-.29.792h.582zm2.637.897h-.26v.888h-.493v-2.383h.982c.5 0 .834.268.834.757 0 .38-.21.621-.543.703l.628.923h-.558zm-.26-1.072v.664h.458c.248 0 .376-.132.376-.33 0-.202-.128-.334-.376-.334zm2.616 1.522h.33c.454 0 .702-.291.702-.753 0-.466-.248-.757-.702-.757h-.33zm-.493.438v-2.383h.842c.722 0 1.176.466 1.176 1.192s-.454 1.191-1.176 1.191z" fill="#fff"/></g></svg>
```

Confirmed this passes checks locally and generally looks alright.
Screenshot from 2021-09-29 19-33-39
Screenshot from 2021-09-29 19-33-12

**Comment by JoostMollie** [comment]
Should be done now @larouxn

**Review by larouxn — APPROVED** [review]

**Comment by larouxn** [comment]
👋  Ran into a few release pipeline issues. Hoping to have this live by end of next week, potentially sooner.

**Comment by larouxn** [comment]
👋 Hello again, your new icons should be live everywhere now.

**Pull request merged**